kornia/augmentation/auto/base.py

```diff
@@ -17,10 +18,20 @@
 class PolicyAugmentBase(ImageSequentialBase):
     """Policy-based image augmentation."""
 
-    def __init__(self, policy: List[SUBPLOLICY_CONFIG]) -> None:
+    def __init__(self, policy: List[SUBPLOLICY_CONFIG], transformation_matrix: str = "silence") -> None:
```

Comment:
```suggestion
    def __init__(self, policy: List[SUBPLOLICY_CONFIG], transformation_matrix_mode: str = "silence") -> None:
```
to avoid confusion with actually passing a transformation matrix